src/shared/symbols.js

```diff
@@ -0,0 +1,15 @@
+'use strict'
+import Symbol from 'es-symbol'
+
+export const ACTION_HANDLER = Symbol('action creator handler')
```

Comment:
but wouldn't you be able to require deep into the compiled source and pull these out?